readme.md

```diff
@@ -38,63 +38,63 @@ $ npm -v
 
 * **Question:** What is the difference between forking and cloning a repository as opposed to just cloning a repository?
 
-* **Your Answer:**
+* **Your Answer:** Forking a repo creates a link between the original repo and your repo that allows you to make a pull request later. Cloning a repo just creates a copy.
 
 ---
 
 - [ ] Run `npm init -y` from the command line
 
 * **Question:** What does `npm init` do? How does the `-y` flag modify that command?
 
-* **Your Answer:**
+* **Your Answer:** npm init initializes npm in your project by creating the package.json file. The y flag answers yes (or fills in the default values) to any prompt that the terminal asks you
 
 ---
 
 - [ ] Take a look at the file that was generated by the previous command
 
 * **Question:** What is the purpose of the following keys? "name", "scripts", "license"
 
-* **Your Answer:**
+* **Your Answer:** The name is the name you give your project. Scripts is the scripts that your project will run. License is which license you are releasing the software under (open source, MIT, etc. ).
 
 ---
 
 - [ ] Create a `.gitignore` file
 
 * **Question:** What is the purpose of the `.gitignore` file? What is the significance of a "dot-file?"
 
-* **Your Answer:**
+* **Your Answer:** A gitignore file tells git to ignore certain files or filetypes so it won't try and commit them to the repo - super helpful for things like thumbs.db and other generated files, API keys, node _modules, private. Any dot-file is generally hidden from the finder/file explorer.  
 
 ---
 
 - [ ] Create an `index.js` file with the following contents: `console.log('Hello, Node!')`
 
 * **Question:** From the command line, how can you run this file?
 
-* **Your Answer:**
+* **Your Answer:** node index.js
 
 ---
 
 - [ ] Run `npm test` from the command line
 
 * **Question:** What happens and how is this related to what is in the `package.json` file? 
 
-* **Your Answer:**
+* **Your Answer:** npm attempts to run the script "test" but since we don't really have one in the package.json we get an error.
```

Comment:
To be more precise, right now the "test" script points towards a command that will throw that error. It's not just that we have no tests.